internal/template/template_test.go

```diff
@@ -91,6 +92,40 @@ func TestMergeData(t *testing.T) {
 	}
 }
 
+func TestCopyData(t *testing.T) {
+	nested := map[string]interface{}{
+		"c": []int{1, 2, 3},
+	}
+	addrNested := fmt.Sprintf("%p", nested)
```

Comment:
was it for your own debugging?

Reply:
nvm saw address comparison below

Reply:
I wanted to extract the address of the example dictionary, and found this to be one way of getting that address